src/app.ts

```diff
@@ -44,7 +53,7 @@ export default class App {
       };
 
       try {
-        await command.execute(message, args, prefix, commandList);
+        await command.execute(message, args, prefix, commandList, this._dbService);
```

Comment:
Not necessarily right now, but maybe we should consider passing some of the arguments into an object? It seems they might continue piling up, and we might not always want to take a lot of unnecessary args in our commands

Reply:
Yeah that was my thinking too, it's a simple approach right now that won't scale so was thinking of enhancing the command creation process when possible.